propolis/src/vmm/mapping.rs

```diff
@@ -545,7 +545,7 @@ mod tests {
 
         let input: u64 = 0xDEADBEEF;
         mapping.as_ref().write(&input).unwrap();
-        let output = mapping.as_ref().read().unwrap();
+        let output: u64 = mapping.as_ref().read().unwrap();
```

Comment:
What changed to need this now?

Reply:
Strange, I saw a compile error previously, but I just removed it and did not see any problem. Reverted in 45f0ee9.